### ISSUE TITLE
Possible to remove files outside the project directory

### ISSUE THREAD

**Issue by mishani0x0ef** [gregnb/filemanager-webpack-plugin#18]
## Config

```
new FileManagerPlugin({
    onStart: {
        delete: "D:\path\to\my\project"
    }
})
```

## Issue
In case if by-mistake in _delete_ function was passed incorrect path - possible unexpected deletion of files (in my case I was able to kill my D-drive). It could be a bit dangerous.

Maybe as solution delete operation could be limited to remove only files and folders that are inside some _root_ folder that should be specified.

## Your Environment

| Tech         | Version |
|--------------|---------|
| filemanager-plugin-webpack  | 1.0.20 |
| node        | 6.9.1 |
| OS      | Windows 10 (version 1703 OS Build 15063.786)  |

**Comment by gregnb** [comment]
@mishani0x0ef Sorry to hear that you wiped out your folder. That sucks. So that I'm understanding correctly 

```
new FileManagerPlugin({
    onStart: {
        delete: "D:\"
    }
})
```

Is what you had for a config?

**Comment by mishani0x0ef** [comment]
@gregnb Yeah, almost. I have passed a string instead of an array like this:

```
new FileManagerPlugin({
    onStart: {
        delete: "D:\path\to\my\project"
    }
})
```

*P.S:* I have recovered my files already and I will be more careful in the future. However, possibility to remove all files makes me surprised.))

**Comment by mishani0x0ef** [comment]
Corrected config in the issue description to make it more clear.

**Comment by gregnb** [comment]
Published in 1.0.21 - Thanks again

**Comment by mishani0x0ef** [comment]
My pleasure. Thanks for the great plugin.
